tox.ini

```diff
@@ -357,6 +357,7 @@ classmethod-decorators =
 
 per-file-ignores =
     test/*: D
+    migrations/*: D
```

Comment:
Why do we want to ignore the migration files in tox?

Reply:
This just ignores the docstring rules.